containers/daap-landing-to-raw/src/var/task/requirements.txt

```diff
@@ -1,2 +1,15 @@
 boto3==1.28.9
 botocore==1.31.9
+attrs==23.1.0
```

Comment:
I had to copy all the base image requirements here, as otherwise they aren't installed by the test job, and they aren't installed in the image.

This problem will be solved if we build a python package for the base image as proposed here https://github.com/ministryofjustice/data-platform/discussions/1613#discussioncomment-7185500